doc/api/cli.md

```diff
@@ -230,6 +230,14 @@ https://github.com/tc39/ecma262/pull/1320.
 
 Both of the above may change in future updates, which will be breaking changes.
 
+### `--heapsnapshot-signal=signal`
+<!-- YAML
+added: REPLACEME
+-->
+
+If set, generate a heap snapshot when the process receives the specified signal.
+Disabled by default.
```

Comment:
docs don't say what valid values for `signal` are - should say accepts full signal names (upper or lower case), and numbers, as well as in the man page below. Also, its done every time signal is received, so perhaps `s/when/each time/`?